ocfweb/context_processors.py

```diff
@@ -3,13 +3,18 @@
 
 from ocflib.lab.hours import DayHours
 
+from ocfweb.component.banner import get_banner_message
+
 
 def ocf_template_processor(request):
     now = datetime.now()
     today = date.today()
     hours = DayHours.from_date(today)
 
+    banner = get_banner_message()
+
     return {
         'lab_is_open': hours.is_open(now),
         'current_lab_hours': hours,
+        'force_lab_closed': banner.visible and banner.force_lab_closed,
```

Comment:
I think all we'd need is
`force_lab_closed: true/false`

You can just set it to false when the lab shouldn't be forced closed.

You're right, you can have the banner but not close the lab. But I don't think you can close the lab without a banner? (Not that we probably need to, but I think it makes sense to be able to)